.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,17 @@
+---
+name: 'Feature Request'
+about: Suggest an idea for this project
+
+---
+
+## Tell us about it
+
+The more specific the better. You can assume you have access to any combination of datasets defined in the schema. When considering an implementation, keep in mind that
+
+- models in ArviZ can have one, or lots of random variables,
```

Comment:
Being picky here. If we're doing a comma I think we need another comma, but only partially sure of this
```suggestion
- models in ArviZ can have one, or lots, of random variables,
```

Reply:
good catch.